SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveMessage.java

```diff
@@ -30,13 +30,13 @@
     // Priority of the message
     protected int priority = 9999;
     // Parent in-app campaign
-    protected SwrveCampaign campaign;
+    protected SwrveMessageCampaign campaign;
```

Comment:
Nice rename of class! Could we rename it to SwrveCampaignMessage and also the corresponding one should be SwrveCampaignConversation? That way they appear beside it each other in file explorer and also in auto complete.